tests/ltp/patches/flistxattr02.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/testcases/kernel/syscalls/flistxattr/flistxattr02.c b/testcases/kernel/syscalls/flistxattr/flistxattr02.c
+index 13aa0b7e7..f1cc48f95 100644
+--- a/testcases/kernel/syscalls/flistxattr/flistxattr02.c
++++ b/testcases/kernel/syscalls/flistxattr/flistxattr02.c
+@@ -17,12 +17,17 @@
+ * 2) flistxattr(2) should return -1 and set errno to EBADF.
+ */
+ 
++/* Currently xattr is not enabled while mounting root file system. Patch is
++ * to mount root file system with xattr enabled and then use it for the test.
++*/
++
++#include <stdio.h>
+ #include "config.h"
+ #include <errno.h>
+ #include <sys/types.h>
+ 
+ #ifdef HAVE_SYS_XATTR_H
+-# include <sys/xattr.h>
```

Comment:
Shouldn't be patched.

Reply:
@letmaik I asked this to be fixed since #include looks more proper than # include. Don't you aggree? Do we really need to take it back?

Reply:
updated as per maik's comments